### DISCUSSION TITLE
Is it possible to monitor Proxmox Backup Server?

### DISCUSSION THREAD

**Discussion by godfreym29** [dougiteixeira/proxmoxve#252] [Feature request] [answered]
I run a PBS server and would like to be able to monitor it 
Is it possible or can this feature be added in the future?

**Comment by dougiteixeira** [comment, marked as answer]
The [`proxmoxer` package](https://pypi.org/project/proxmoxer/) is compatible with PBS ([the API is documented here](https://pbs.proxmox.com/docs/api-viewer/index.html)). Although it is possible to include this, it is not in the plans.

Maybe it would be better to build a new integration to monitor it.
